doc/api/stream.md

```diff
@@ -2118,6 +2118,45 @@ import { Readable } from 'stream';
 await Readable.from([1, 2, 3, 4]).take(2).toArray(); // [1, 2]
 ```
 
+### `readable.reduce(fn[, initial[, options])`
```

Comment:
```suggestion
### `readable.reduce(fn[, initial[, options]])`
```

To align with what the code currently supports. What if you want to `reduce()` with options but without an initial value?

Reply:
An ugly solution would be `reduce(fn, undefined, options)` and changing the spec from ["initialValue is not present"](https://tc39.es/proposal-iterator-helpers/#sec-iteratorprototype.reduce) to "initialValue is not undefined".

Reply:
I think that would also be a hard sell because it diverges from how Array.prototype.reduce behaves:

```js
[1,2,3].reduce((p, c) => p + c); // 6
[1,2,3].reduce((p, c) => p + c, undefined); // NaN
```

Reply:
@devsnek thoughts?

Reply:
@benjamingr ping

Reply:
Ah, didn't realize this was blocking the review, fixed.